README.md

```diff
@@ -133,6 +133,10 @@ from baybe.searchspace import SearchSpace
 searchspace = SearchSpace.from_product(parameters)
 ```
 
+See the [search spaces section](https://emdgroup.github.io/baybe/userguide/searchspace.html)

```

Comment:
search space instead of search spaces

Reply:
I think this should then be consistent on the entire page. Right now, we sometimes have plural, sometimes singular. What do you (@AVHopp, @Scienfitz) prefer?

Reply:
I do not have a strong opinion here, just wanted to make it consistent to the name of the user guide, but this is not super urgent imo, feel free to also just resolve and ignore.

Reply:
now plural everywhere, which makes it consistent with the sidebar of our user guide